Components/AccordionVertical/style.styl

```diff
@@ -0,0 +1,69 @@
+$rupture.scale = lookup('$globalRuptureScale') || (0 575px 767px 991px 1199px)
+$rupture.anti-overlap = lookup('$globalRuptureAntiOverlap') || -1px
+$containerPadding = lookup('$globalContainerPadding') || 15px
+$containerMaxWidth = lookup('$globalContainerMaxWidth') || 1140px
+$colorBrand = lookup('$globalColorBrand') || #254fc7
+$buttonBackgroundColor = #f3f3f3
```

Comment:
I don't mind so much as long as we're consistent. My main reasoning was that I could not see it being common to use the main brand colour for the background colour of the accordion buttons, and I also don't want to make this component harder to quickly re-use by adding another global variable.